jobs/build-publish-image.groovy

```diff
@@ -38,6 +38,9 @@ someone else's publish-deploy on purpose.""",
 currentBuild.displayName = "${currentBuild.displayName} (${params.GIT_REVISION})";
 
 def runScript() {
+   // Prune docker images before building if under 1.5GB of disk space.
+   sh("[ $(df -BM --output=avail . | tr -cd 0-9) -gt 1500 ] || docker image prune -af")
```

Comment:
Probably want to do this after the sync, since the sync could eat up disk-space.  Otherwise looks great!